packages/nodes-base/nodes/Twist/CommentDescription.ts

```diff
@@ -0,0 +1,623 @@
+					'comment',
+				],
+				operation: [
+					'add',
+				],
+			},
+		},
+		options: [
+						{
+				displayName: 'Actions',
+				name: 'actionsUi',
+				type: 'fixedCollection',
+				placeholder: 'Add Action',
+				typeOptions: {
+					multipleValues: true,
+				},
+				options: [
+					{
+						displayName: 'Action',
+						name: 'actionValues',
+						values: [
+							{
+								displayName: 'Action',
+								name: 'action',
+								type: 'options',
+								description: 'The action of the button',
+								options: [
+									{
+										name: 'Open URL',
+										value: 'open_url',
+									},
+									{
+										name: 'Prefill Message',
+										value: 'prefill_message',
+									},
+									{
+										name: 'Send Reply',
+										value: 'send_reply',
+									},
+								],
+								default: '',
+							},
+							{
+								displayName: 'Button Text',
+								name: 'button_text',
+								type: 'string',
+								description: 'The text for the action button.',
+								default: '',
+							},
+							{
+								displayName: 'Message',
+								name: 'message',
+								type: 'string',
+								displayOptions: {
+									show: {
+										action: [
+											'send_reply',
+											'prefill_message',
+										],
+									},
+								},
+								description: 'The text for the action button.',
+								default: '',
+							},
+							{
+								displayName: 'Type',
+								name: 'type',
+								type: 'options',
+								description: 'The type of the button, for now just action is available.',
+								options: [
+									{
+										name: 'Action',
+										value: 'action',
+									},
+								],
+								default: '',
+							},
+							{
+								displayName: 'URL',
+								name: 'url',
+								type: 'string',
+								displayOptions: {
+									show: {
+										action: [
+											'open_url',
+										],
+									},
+								},
+								description: 'URL to redirect',
+								default: '',
+							},
+						],
+					},
+				],
+			},
+			{
+				displayName: 'Attachments',
+				name: 'binaryProperties',
+				type: 'string',
+				default: 'data',
+				description: 'Name of the property that holds the binary data. Multiple can be defined separated by comma.',
+			},
+			{
+				displayName: 'Direct Mentions',
+				name: 'direct_mentions',
+				type: 'multiOptions',
+				typeOptions: {
+					loadOptionsMethod: 'getUsers',
+										loadOptionsDependsOn: [
+						'workspaceId',
+					],
+				},
+				default: [],
+				description: `The users that are directly mentioned`,
+			},
+			{
+				displayName: 'Recipients',
+				name: 'recipients',
+				type: 'multiOptions',
+				typeOptions: {
+					loadOptionsMethod: 'getUsers',
+					loadOptionsDependsOn: [
+						'workspaceId',
+					],
+				},
+				default: [],
+				description: 'The users that will attached to the comment.',
+			},
+						{
+				displayName: 'Temporary ID',
+				name: 'temp_id',
+				type: 'number',
+				default: 0,
```

Comment:
Most IDs are usually strings - wondering if `temp_id` should be a string or number. If already checked, please disregard.